controller/src/controller/models/domain/rsptag.py

```diff
@@ -157,6 +169,7 @@ def alias(cls, tag: str) -> Self:
             version=None,
             cycle=cycle,
             display_name=display_name,
+            date=None,
```

Comment:
Oh, this is a helpful reminder: `RSPImage.resolve_alias` should also set the `date` field on the alias tag if it resolves to a tag with a `date` field that is not `None`, similar to how we override the cycle when we resolve an alias tag. (And that should have a test as well.)

Reply:
While I agree, it also doesn't matter because alias tags are not subject to filtering; the only reason to create aliases is in order to pin things to the menu, after all.